### ISSUE TITLE
Can't access role teacher and student - ERROR Livewire encountered a missing root tag when trying to render a component. When rendering a Blade view, make sure it contains a root HTML tag.

### ISSUE THREAD

**Issue by saddadnabbil** [yungifez/skuul#363]
![image](https://github.com/yungifez/skuul/assets/77654396/2cfc8ba0-6f17-455b-9181-6b72dd25c783)
when i login as super admin and admin it's working, but when i login as teacher and student it's showing error 

```
User
Found the following N+1 queries in this request:

Model: App\Models\School => Relation: App\Models\ClassGroup - You should add "with('App\Models\MyClass')" to eager-load this relation.
Model: App\Models\School => Relation: myClasses - You should add "with('myClasses')" to eager-load this relation.
Model: App\Models\MyClass => Relation: App\Models\Subject - You should add "with('App\Models\Subject')" to eager-load this relation.
Model: App\Models\MyClass => Relation: App\Models\Section - You should add "with('App\Models\Section')" to eager-load this relation.
```

**Comment by yungifez** [comment]
I'll investigate those

Thank you for reporting this

**Comment by saddadnabbil** [comment]
thank you! please inform to me if it's done

**Comment by yungifez** [comment]
This issue has been fixed in v2.6.2

If you followed the git tag installation method, updating is as simple as running`php artisan skuul:update`

If not, just git pull
